### DISCUSSION TITLE
INSERT HL7 MESSAGE DATA TO DATABASE

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5469] [Q&A] [answered]
**Good day
I need to insert HL7 message data to a Database.
I receive an hl7 as a json blob since within the json message not only the hl7 message comes, other data also comes.
I need to have an idea of how to start to do it, to know if it is necessary to serialize the message, etc... in the background, to have a sequential action plan.**

![image](https://user-images.githubusercontent.com/98954172/195628407-f9e826d4-64b6-435c-b7ad-ecb1f97ae9d7.png)

Thank you very much for the help :)

**Comment by pacmano1** [comment, marked as answer]
If you mean "I want to insert the JSON field that has hl7 in it as shown in my screenshot to a database",  then insert it as it is.

You will serialize it when you pull it back out if you need to use it as HL7.

The forum and github has examples of DB inserts.
